kano_updater/scenarios.py

```diff
@@ -965,4 +977,10 @@ def beta_3_14_0_to_beta_3_14_1(self):
         pass
 
     def beta_3_14_1_to_beta_3_15_0(self):
-        pass
+        try:
+            import shutil
+            import os
+
+            shutil.copy(PIP_CONF, PYDIST_PATH)
```

Comment:
Is `PYDIST_PATH` defined anywhere?

Reply:
Thanks, it's defined in `paths` as `PYLIBS_DIR`, corrected.